xbmc/pvr/timers/PVRTimers.cpp

```diff
@@ -248,6 +248,47 @@ bool CPVRTimers::UpdateEntries(const CPVRTimers &timers)
     UpdateEpgEvent(*timerIt);
   }
 
+  /* go through the timer list and check for parent timers */
+  for (MapTags::const_iterator it = timers.m_tags.begin(); it != timers.m_tags.end(); ++it)
```

Comment:
c++11: <code>for (const auto &tagsEntry : m_tags)</code> - much more elegant, imo.